package.json

```diff
@@ -65,7 +65,8 @@
     "nth-check": "^2.0.1",
     "@typescript-eslint/eslint-plugin": "^6.14.0",
     "@typescript-eslint/parser": "^6.14.0",
-    "fork-ts-checker-webpack-plugin": "^6.5.3"
+    "fork-ts-checker-webpack-plugin": "^6.5.3",
+    "rxjs": "^7.8.1"
```

Comment:
Is redux observable pulling another lower version? Why is this required?

Reply:
I was getting some errors due to '@craftercms/content' having rxjs v7.0.0. Should we update the sdk? and or keep this resolution?

Reply:
Update sdk